### PR TITLE
Add Microsoft Sentinel

### PR DESCRIPTION
**Pull request by kfriede**
https://github.com/kfriede/awesome-microsoft-sentinel

Resources for Microsoft Azure Sentinel and related KQL, SIEM, and SOAR functionality 

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- [x] Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- [x] **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- [x] **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
    - https://github.com/sindresorhus/awesome/pull/2736
    - https://github.com/sindresorhus/awesome/pull/2647
- [x] You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- [x] This pull request has a title in the format `Add Name of List`. It should not contain the word `Awesome`.
- [x] Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
- [x] Your entry should be added at the bottom of the appropriate category.
- [x] The title of your entry should be title-cased and the URL to your list should end in `#readme`.
- [x] No blockchain-related lists.
- [x] The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- [x] Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- [x] The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- [x] It's the result of hard work and the best I could possibly produce.
- [x] The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
- [x] The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
- [x] Non-generated Markdown file in a GitHub repo.
- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- [x] Not a duplicate. Please search for existing submissions.
- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
- [x] Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [x] Includes a project logo/illustration whenever possible.
- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
- [x] Has a Table of Contents section.
- [x] Has an appropriate license.
- [x] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
- [x] All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- [x] Has consistent formatting and proper spelling/grammar.
- [x] Does not use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- [x] Does not include a CI (e.g. GitHub Actions) badge.<br>You can still use a CI for linting, but the badge has no value in the readme.
- [x] Does not include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by kfriede** [comment]
@sindresorhus I went through line-by-line when I checked everything off. Would you care to inform me what I'm missing so I can fix it?

**Review by The-Debarghya — APPROVED** [review]

**Comment by bdalalhaljwysm** [comment]
def fibonacci(n)
  return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
end

**Comment by kfriede** [comment]
unicorn 🦄

**Comment by donBarbos** [comment]
It seems [this](https://github.com/reprise99/awesome-kql-sentinel) and [this](https://github.com/cylaris/awesomekql) repositories already covers your topic

Also, your repository may not fit these criteria:
```markdown
- [ ] It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
```

**Review by iamhonest1 — COMMENTED** [review]
✌️

File: readme.md [hunk 1]
```diff
@@ -640,6 +640,7 @@
 - [Password Cracking](https://github.com/n0kovo/awesome-password-cracking#readme) - The process of recovering passwords from data that has been stored in or transmitted by a system in scrambled form.
```
Comment (Gnssrl) [inline readme.md:640]: Ho

**Review by Gnssrl — CHANGES_REQUESTED** [review]

**Comment by infosecB** [comment]
- Recommend renaming repo to "awesome-azure-sentinel" or "awesome-microsoft-sentinel"
- Pull request title should match name of repo (e.g. without above change, "Add Sentinel")

**Comment by kfriede** [comment]
> * Recommend renaming repo to "awesome-azure-sentinel" or "awesome-microsoft-sentinel"
> * Pull request title should match name of repo (e.g. without above change, "Add Sentinel")

Updated repository name, no PR name change required (since repository name now matches PR)

**Comment by kfriede** [comment]
unicorn 🦄

**Review by emmanuelgautier — COMMENTED** [review]
Nice list.

You have some missing punctuation warnings here : https://github.com/kfriede/awesome-microsoft-sentinel/actions/runs/6644291312/job/18053167510#step:3:10

File: readme.md [hunk 2]
```diff
@@ -643,6 +643,7 @@
...
 - [Security Card Games](https://github.com/Karneades/awesome-security-card-games#readme) - Train your skills and discuss various security topics.
 - [Suricata](https://github.com/satta/awesome-suricata#readme) - Intrusion detection/prevention system and network security monitoring engine.
+- [Microsoft Sentinel](https://github.com/kfriede/awesome-microsoft-sentinel#readme) - Microsoft Sentinel cloud-native SIEM/SOAR solution.
```
Comment (sindresorhus) [inline readme.md:646]: The description should not contain the list title. Also not clear what SIEM/SOAR is. Needs a better description in general.

**Comment by sindresorhus** [comment]
The list is a bit thin on content.

**Comment by sindresorhus** [comment]
It's also not following multiple of the guidelines, like `.` at the end of descriptions. I suggest working on it some more and try submitting it again at a later point (at least wait 6 months).

**Pull request closed without merging**